src/circuit/utils.ts

```diff
@@ -115,6 +115,7 @@ export async function namespaceToCid (namespace: string): Promise<CID> {
 /**
  * returns number of ms between now and expiration time
  */
-export function getExpiration (expireTime: bigint): number {
-  return Number(expireTime) - new Date().getTime()
+export function getExpirationSeconds (expireTime: bigint): number {
```

Comment:
The comment still says it returns the number of ms but the function has been renamed to say it returns seconds.